types/react-tracking/tslint.json

```diff
@@ -2,6 +2,7 @@
     "extends": "dtslint/dt.json",
     "rules": {
         "ban-types": false,
-        "no-unnecessary-generics": false
+        "no-unnecessary-generics": false,
+        "no-outside-dependencies": false
```

Comment:
Don't disable this lint rule - instead make a PR to https://github.com/microsoft/DefinitelyTyped-tools/tree/master/packages/definitions-parser/allowedPackageJsonDependencies.txt

Reply:
Although I don't see any new imports, so I don't see why it was added at all?

Reply:
Maybe it only fails on my machine, I can make a commit to revert it and test if you think it might work.

Reply:
Just pushed the revert, will resolve this conversation if the CI passes.